mungegithub/pulls/lgtm_after_commit.go

```diff
@@ -72,6 +72,11 @@ func (LGTMAfterCommitMunger) MungePullRequest(config *config.MungeConfig, pr *gi
 		return
 	}
 
+	// Don't remove the LGTM from merged PRs
+	if pr.Merged != nil && *pr.Merged {
```

Comment:
This shouldn't happen, right?  We should abort way back up in the ForEachIssueGet() if the second Get() didn't give us an "open" issue...